tests/utils_tests/test_datastructures.py

```diff
@@ -34,6 +35,12 @@ def test_discard(self):
         s.discard(2)
         self.assertEqual(len(s), 1)
 
+    def test_reversed(self):
+        s = OrderedSet([1, 2, 3])
+        self.assertEqual(list(reversed(s)), [3, 2, 1])
```

Comment:
Before this line, you can add `s = reversed(s)`, and then check `s` in the following lines instead of `reversed(s)`.

Reply:
That's cleaner, thanks!